File: Udemy-Neal-Davis/section-12-database-analytics/hol-198-elasticache-cluster/main.py
import redis

primary_endpoint="master.simplerediscluster.sua2pq.euw2.cache.amazonaws.com"
cache = redis.StrictRedis(host=primary_endpoint, port=6379, db=0)

cache.set('name', 'Tochukwu')
cache.set('role', 'Staff Engineer')

value = cache.get('role')

# Decoding the bytes literal to a string
decoded_value = value.decode('utf-8')
print(decoded_value)

File: Udemy-Neal-Davis/section-12-database-analytics/hol-198-elasticache-cluster/main.js
const Redis = require("ioredis");

const client = new Redis({
  host: "master.simplerediscluster.sua2pq.euw2.cache.amazonaws.com",
  port: 6379,
  //   port: 6380,
  //   tls: {}, // Enables TLS
});

(async () => {
  // Set and assert
  const setResult = await client.set("key", "value");
  console.assert(setResult === "OK");

  // Get and assert
  const getResult = await client.get("key");
  console.assert(getResult === "value");

  // Close the connection
  client.disconnect();
})();
